SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function parseProductTypeNSXMgrConfiguration {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
+	$NSXMgrConfigTypeValues =  New-Object -TypeName psobject
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "passwdMinimumLength" -notepropertyvalue "Default" 
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "apiPasswdMaxFailAttempts" -notepropertyvalue "Default"
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "apiPasswdMaxFailIntervalInSec" -notepropertyvalue "Default"
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "apiPasswdUnlockIntervalInSec" -notepropertyvalue "Default"
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "cliPasswdMaxFailAttempts" -notepropertyvalue "Default"
+	$NSXMgrConfigTypeValues | Add-Member -notepropertyname "cliPasswdMaxFailIntervalInSec" -notepropertyvalue "Default"
+	foreach ($configuration in $productConfiguration.NSXMgr.PSObject.Properties) {
+		if ($NSXMgrConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			$NSXMgrConfigTypeValues.$($configuration.Name) = $configuration.Value
+		} else {
+			Write-LogMessage -Type ERROR -Message "parameter ("$configuration.Name") not found (NSXEdge)" -Colour Red
+		}
+	}
+	foreach ($configuration in $productConfiguration.Default.PSObject.Properties) {
+		if ($NSXMgrConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			if ($NSXMgrConfigTypeValues.$($configuration.Name) -eq "Default") {
+				$NSXMgrConfigTypeValues.$($configuration.Name) = $configuration.Value
+			}
+		}
+		if ($NSXMgrConfigTypeValues.psobject.properties.match($("api"+$($configuration.Name))).Count) {
+			if ($NSXMgrConfigTypeValues.$($("api"+$($configuration.Name))) -eq "Default") {
+				$NSXMgrConfigTypeValues.$($("api"+$($configuration.Name))) = $configuration.Value
+			}
+		}
+		if ($NSXMgrConfigTypeValues.psobject.properties.match($("cli"+$($configuration.Name))).Count) {
+			if ($NSXMgrConfigTypeValues.$($("cli"+$($configuration.Name))) -eq "Default") {
+				$NSXMgrConfigTypeValues.$($("cli"+$($configuration.Name))) = $configuration.Value
+			}
+		}
+	}
+	$standardConfigValue = New-Object -TypeName psobject
+	$standardConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "NSXMgr"
+	$standardConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $NSXMgrConfigTypeValues
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function parseProductTypeNSXEdgeConfiguration {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
```

Comment:
Add blank line after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175